src/Streamly/Internal/Data/Stream/IsStream/Generate.hs

```diff
@@ -529,21 +542,18 @@ iterateMSerial step = fromStreamS . S.iterateM step
 -- represents the tuple @(x,y)@ returned by it in a given iteration. We define
 -- the first element of the tuple in terms of the second.
 --
--- @
--- import Streamly.Internal.Data.Stream.IsStream as Stream
--- import System.IO.Unsafe (unsafeInterleaveIO)
---
--- main = do
---     Stream.mapM_ print $ Stream.mfix f
+-- >>> import Streamly.Internal.Data.Stream.IsStream as Stream
+-- >>> import System.IO.Unsafe (unsafeInterleaveIO)
```

Comment:
These import can go to $setup

Reply:
First one can go but the second one is very specific to this code so better placed here. The setup section is usually not visible to users which makes those imports a bit obscure.